src/sentry/api/endpoints/team_project_index.py

```diff
@@ -112,14 +116,22 @@ def post(self, request, team):
                         name=result['name'],
                         slug=result.get('slug'),
                         organization=team.organization,
-                        team=team
+                        team=team,
+                        platform=result.get('platform')
                     )
             except IntegrityError:
                 return Response(
                     {'detail': 'A project with this slug already exists.'},
                     status=409,
                 )
 
+            platform = result.get('platform')
+            if platform:
```

Comment:
Got it-good point. We discussed and decided to take it out and revisit later if needed.